NGit.Test/NGit.Api/ApplyCommandTest.cs

```diff
@@ -204,5 +204,29 @@ public virtual void TestModifyZ()
 				()[0]);
 			CheckFile(new FilePath(db.WorkTree, "Z"), b.GetString(0, b.Size(), false));
 		}
+
+        [Test]
+        public void ApplyingPatchWithInconsistentMacLineEndingsIsSuccessfullyApplied()
+        {
+            const string pre = "a\r\r\nb";
+            const string post = "a\r\nb";
+            var git = new Git(db);
+            var stream = new MemoryStream();
+            var modifiedFile = Path.Combine(db.WorkTree, "file.txt");
+
+            File.WriteAllText(modifiedFile, pre);
+            git.Add().AddFilepattern(".").Call();
+            git.Commit().SetMessage("Initial commit").Call();
+
+            File.WriteAllText(modifiedFile, post);
+            git.Diff().SetOutputStream(stream).Call();
+
+            stream.Position = 0;
+            File.WriteAllText(modifiedFile, pre);
+
+            git.Apply().SetPatch(stream).Call();
+
+            Assert.That(File.ReadAllText(modifiedFile), Is.EqualTo(post));
+        }
```

Comment:
This test is not following the pattern that all other tests in the suite seems to follow (having a preImage, postImage and a patch in the NGit.Diff and later do assertions on the ApplyResult). This test runs 4 times but m_UseCrlfPatches and m_UseCrlfFiles are ignored because we do not call Init. Any reason why we can't make this test following the same patters of the others?